voltixApp/VoltixApp/View Models/KeysignViewModel.swift

```diff
@@ -65,7 +65,12 @@ class KeysignViewModel: ObservableObject {
         self.vault = vault
         self.keysignPayload = keysignPayload
     }
-    
+    func getTransactionExplorerURL(txid: String) -> String{
+        if let keysignPayload {
```

Comment:
guard would be a better check in this case.

Reply:
fixed